splitgpg2/test_server.py

```diff
@@ -732,6 +732,28 @@ def test_003_option_typo(self) -> None:
             mock.call.warning('Unsupported config option: %s', 'no_such_option'),
         )
 
+    def test_004_multiple_files(self) -> None:
```

Comment:
This test doesn't really cover changes in the load_config_files() function. Maybe you can test it by mocking `xdg.BaseDirectory.xdg_config_home` with a temp dir and put test configs there?